e2e-tests/recover/seedPhrase.spec.js

```diff
@@ -0,0 +1,44 @@
+const { createTestSubaccount } = require("../utils/account");
+
+describe("Account Recovery Using Seed Phrase", () => {
+    let testAccount;
+    beforeAll(async () => {
+        testAccount = await createTestSubaccount();
+    });
+
+    afterAll(async () => {
+        await browser.close();
+        await testAccount.delete();
+    });
+
+    it("navigates to seed phrase page successfully", async () => {
+        const user = await page.goto("/").then(() => page.getDocument());
```

Comment:
I'm not a fan of this global `page` object -- this won't scale as our tests become more complex and I'd like to avoid needing to refactor these later.

There can be multiple `page` objects per browser context.  Would like for us to use Page Object Models (POMs) for everything so that the dependency on a specific page is explicit -- in short, there shouldn't be  global variables for things that we can have multiple of.

I actually couldn't even find mention of a `page` global anywhere in the Playwright docs; is this part of one of the extensions? :hocho: We want to err on the side of 'flexible' over 'terse' when it comes to Playwright usage; it looks like default `page` and `context` are passed into `describe` and `test` closures for us explicitly.  Was this new global part of wrapping the test runner with Jest or something similar?

Basically, I want us to be following this pattern:
https://playwright.dev/docs/test-pom/
So that we can re-use the same POM objects in multiple test suites; I want the suites themselves to only be 'create POM objects, call methods on those POM objects, then assert against the POM object.  Ideally all of the page-specific selector logic etc. is encapsulated inside of named methods on the POM.

Ideally I'd like to avoid adding wrappers around `playwright-test` until we've validated that we need them; it makes it harder to reason about the API when it is split up across multiple wrapping toolsets.  If we _do_ need an added wrapper around the playwright test runner, we'll need to document that dependency and reference its documentation somewhere (README.md?) so that other engineers on the team can find documentation about these efficiently.

Reply:
This global is indeed exposed by the `jest-playwright` runner. `jest-playwright` exposes [the following globals](https://github.com/playwright-community/jest-playwright#globals). I don't think this will cause scaling issues though since each test file gets a new instance of `page` and `context`.

I like the POM pattern but this should not impact which test runner we decide to go with since we can use this pattern with any test runner.